src/query.md

```diff
@@ -180,20 +180,14 @@ the big macro invocation in
 
 ```rust,ignore
 rustc_queries! {
-    Other {
-        /// Records the type of every item.
-        query type_of(key: DefId) -> Ty<'tcx> {
-            cache { key.is_local() }
-        }
+    /// Records the type of every item.
```

Comment:
can we mention that you should add a doc-comment, or else it shames you about it in the generated docs lmao
https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/context/struct.TyCtxt.html#method.trigger_delay_span_bug

Reply:
We should probably also move the shaming from rustdoc to shaming you with a compiler instead at some point

Reply:
https://github.com/rust-lang/rust/issues/104008